diff --git a/cmd/grv/vendor/gopkg.in/libgit2/git2go.v27/script/build-libgit2-static.sh b/cmd/grv/vendor/gopkg.in/libgit2/git2go.v27/script/build-libgit2-static.sh
index 5e01b53..433cb9f 100755
--- a/cmd/grv/vendor/gopkg.in/libgit2/git2go.v27/script/build-libgit2-static.sh
+++ b/cmd/grv/vendor/gopkg.in/libgit2/git2go.v27/script/build-libgit2-static.sh
@@ -13,6 +13,7 @@ cmake -DTHREADSAFE=ON \
       -DBUILD_SHARED_LIBS=OFF \
       -DCMAKE_C_FLAGS=-fPIC \
       -DCMAKE_BUILD_TYPE="RelWithDebInfo" \
+      -DCMAKE_PREFIX_PATH=${PREFIX} \
       -DCMAKE_INSTALL_PREFIX=../install \
       -DUSE_EXT_HTTP_PARSER=OFF \
       -DUSE_HTTPS=OFF \
